cmake/packaging.cmake

```diff
@@ -0,0 +1,34 @@
+set(CPACK_PACKAGE_NAME "cbmc") # XXX should this be cprover instead?
+set(CPACK_PACKAGE_VENDOR "Diffblue Ltd.")
+set(CPACK_PACKAGE_CONTACT "kroening@cs.ox.ac.uk") # XXX should this be a Diffblue email address instead?
```

Comment:
Clarification on this point would be great, this is the address used in in `src/cbmc/dist-linux`, the current Ubuntu package lists “Ubuntu Developers” as the maintainer and @tautschnig as the ‘Original Maintainer’, so I’m genuinely unsure what to put here.

Reply:
Can you put multiple contacts in? Might also be worth asking @peterschrammel for an opinion too.

Reply:
If possible I'd put `info@diffblue.com` to have a stable address.